Plugins/Statistics/GitStatistics/FormGitStatistics.cs

```diff
@@ -74,7 +74,7 @@ public FormGitStatistics(IGitModule module, string codeFilePattern, bool countSu
             splitContainer7.SplitterDistance = DpiUtil.Scale(splitContainer7.SplitterDistance);
             splitContainer8.SplitterDistance = DpiUtil.Scale(splitContainer8.SplitterDistance);
 
-            TotalLinesOfCode.Font = new Font(SystemFonts.MessageBoxFont.FontFamily, 9.75F, FontStyle.Bold, GraphicsUnit.Point);
+            TotalLinesOfCode.Font = new Font(TotalLinesOfCode.Font, FontStyle.Bold);
```

Comment:
Ditto.

Reply:
Thank you for all designer-related comments, this instance is fine as it is not in auto-generated section.